Update python brasil 2023 (#27)
pythonbrasil/manual: source/organizacao/index.md

- Orçar e fechar o Local do evento
- Orçar e fechar Internet
- Orçar e fechar Fotografia
- Organizar Transmissão online
- Orçar e fechar Tradução Simultanea
- Orçar e fechar Comunicação no dia do evento
- Orçar e fechar time de Libras

####  Time de Experiencia do evento
O time de experiencia do usuario deve conseguir integrar as pessoas e mostrar o carinho que a cidade sede quer transmitir, tentando organizar atividades espaços e experiencias para as pessoas que vem ao evento.
- Coffe Breack
  - organizar coffe vegano
  - organizar coffe vegetariano
  - organizar coffe onivaro
  - organizar coffe celiaco
- Kit banheiro
- Lighttalk
- Guarda Volumes
- Coworking
- Sleep/Relax space
- Guia da cidade

####  Time de Administração
O time administrativo, deve conter principalmente pessoas de tomada de descisão, fazendo o papel de contratação orçamento e cuidado com fornecedores, são bem vindas pessoas com garagens grandes.
- Orçamentos
- Contratos
- Imobilizado do evento:
  - cordinhas
  - crachas
  - camisas
  - brindes
  - etc....
  - 

> ####  Time de Tecnico

Reviewer: Esse time ficou um pouco confuso pra mim quais as atribuições dele.
Cuidar das atualizações do site acho que poderia ser >> Criar o site do evento utilizando a URL que a Apyb disponibiliza (aquela 2022.pyhtonbrasil, 2023.pythonbrasil, etc) com base na identidade visual criada pelo time de design.
(outro topico sugerido pra adicionar) Manter o site atualizado com todas as informações que quisermos passar ao público (keynotes, plano de patrocinio, disponibilização da grade de palestras, local do evento, etc)

--
Cuidar dos acessos acho dificil centralizar em um time visto que tem um monte de sistema diferente e boa parte quem cuida é a apyb se n me engano.

--
Sistema de notificação n entendi.

O time tecnico é de longe talvez o que mais pessoas va querer participar pois é um evento de tecnologia. Mas esse time tem a responsabilidade de:
- Cuidar das atualizações do site
- Cuidar dos acessos (senhas, tokens, sistemas)
- Criar sistemas de notificação e auxilio aos demais times.

>     

Reviewer: Eu adicionaria um time de conteúdo ou qq outro nome relacionado. Aqui uma sugestão de responsabilidades

```suggestion
 #### Time de Conteúdo
 Responsável por gerenciar a grade de atividades oficiais do evento como palestras, tutoriais, assembleias, keynotes.
    - Criação e configuração do evento na ferramenta de submissão de atividades (por padrão utilizamos a ferramenta pretalx)
    - Criação do formulário de submissão de atividades considerando apenas aquelas que as pessoas possam vir a submeter.
    - Avaliação de atividades para definir quais entram na grade
    - Criação e configuração do evento na ferramenta de ingressos (por padrão utilizamos o eventbrite)
    - Porta-voz de relacionamento com os keynotes
```

## Gestão e organização

Uma vez feito a descrição dos times defina uma agenda mensal/quinzenal/semanal conforme o evento se aproxima. Irei sugerir aqui uma ordem de trabalho mas não existe regra essa ordem esta sempre em discução e deve ser sempre atualizada:

Atividades:
 - Definir Times

>  - Definir Identidade Visual com principais elementos e ja com mockups. Alguns mockups uteis:

Reviewer: Eu colocaria como primeiros itens e principais a criação do plano dde patrocinio ja com a identidade do evento e a criação do design do site, em terceiro os templates pra publicacao. O resto é menos prioritario, mas nao menos importante.

   - Cracha
   - Camisa
   - Cordinhas
   - Templates para publicação
     - palestrantes
     - keynotes
     - patrocinios
     - avisos
 - Definir plano de patrocinio:
   - Beneficios disponiveis
     - fisicos 
     - digitais
   - Cotas
   - Estrategia de comunicação (2023 adotamos evento educacional)
 - Definir Data e local e grade temporaria:
   - Grade temporaria seria um planejamento de slots quantos keynotes, quantas palestras, horarios, numeros de tutorias, dias de sprint, dias de tutorial
   - o local para os tutorias devem incluir laboratorios de informatica.

>    - o local para as palestras seria bom contemplar 

Reviewer: **Necessário** contemplar tudo isso de baixo po kk.
Banheiros essencial, acessibilidade msm coisa pq vimos que pelo menos alguém cm pé quebrado sempre tem na pybr, sala pra org/keynotes a gente n vive sem, espaço pro coffe tbm mto importante.

- Orçar e fechar Tradução Simultanea
- Orçar e fechar Comunicação no dia do evento 
  - Radios ou um canal rapido de conversa para o operacional
- Orçar e fechar time de Libras
- Organizar transmissão
  - Chaves do youtube 
  - links para as transmissões
  - programação de start e stop das trasnmissões
- Buscar locais para realização dos tutoriais

####  Time de Experiencia do evento
O time de experiencia do usuario deve conseguir integrar as pessoas e mostrar o carinho que a cidade sede quer transmitir, tentando organizar atividades espaços e experiencias para as pessoas que vem ao evento.
- Coffe Breack
  - organizar coffe vegano
  - organizar coffe vegetariano
  - organizar coffe onivaro
  - organizar coffe celiaco
- Kit banheiro
- Lighttalk
- Guarda Volumes
- Coworking
- Sleep/Relax space
- Guia da cidade

####  Time de Administração
O time administrativo, deve conter principalmente pessoas de tomada de descisão, fazendo o papel de contratação orçamento e cuidado com fornecedores, são bem vindas pessoas com garagens grandes.
- Orçamentos
- Contratos
- Imobilizado do evento:
  - cordinhas
  - crachas
  - camisas
  - brindes
  - etc....
  - 

####  Time de Tecnico

> O time tecnico é de longe talvez o que mais pessoas va querer participar pois é um evento de tecnologia. Mas esse time tem a responsabilidade de:

Reviewer 2: Muito importante adicionar que o site da Python Brasil deve ser estático, independente de qual ferramenta queiram usar

- Criar o site do evento utilizando a URL que a Apyb disponibiliza (aquela 2022.pyhtonbrasil, 2023.pythonbrasil, etc) com base na identidade visual criada pelo time de design.
- Manter o site atualizado com todas as informações que forem passadas ao público (keynotes, plano de patrocinio, disponibilização da grade de palestras, local do evento, etc)
- Cuidar dos acessos (Autorizar acessos a arquivos, sistemas e repositorios)
- Criar sistemas de notificação, criando formas de facilitar para que os participantes possam facilmente receber noticias do evento
  - Aplicativo
  - Canais em chats 
  - Notificar atualizações do site 
- Criar integração com plataforma de ingressos e palestras.
  - links ou outras integrações

> - Auxilio aos demais times.

Reviewer 2: Isso é uma coisa meio vaga, então não sei se vale a pena especificar

#### Time de Conteúdo

Responsável por gerenciar a grade de atividades oficiais do evento como palestras, tutoriais, assembleias, keynotes.
- Criação e configuração do evento na ferramenta de submissão de atividades (por padrão utilizamos a ferramenta pretalx)

> - Criação do formulário de submissão de atividades considerando apenas aquelas que as pessoas possam vir a submeter.

Reviewer 2: Essa frase está um pouco confusa pra mim, poderia especificar melhor, por favor?

Reviewer: @anadulce essa fui eu que escrevi nas pressas hahaha eu quis dizer que era criar o formulario de submissao só pra palestras tutoriais e sprints pq no final a gente sempre adiciona um monte de tema no pretalx assembleia, keynote etc.
Mas da pra simplificar a frase pra:
- Criação do formulário de submissão de atividades na ferramenta escolhida

- Avaliação de atividades para definir quais entram na grade

> - Criação e configuração do evento na ferramenta de ingressos (por padrão utilizamos o eventbrite)

Reviewer 2: Eu acho que isso são coisas diferentes, uma é ingresso outra é conteúdo

Reviewer: Achei as informações mto boas, mas quis dizer um tópico ou um time separado pra cuidar de ingressos?

Reviewer 2: Ingressos pode ficar no time do financeiro, não faz sentido unir ingresso e conteúdo, porque são coisas completamente diferentes.
Como tem informações muito específicas, acho que valeria um tópico separado sim.

Reviewer 2: Outro ponto válido aqui é deliberar sobre tópicos que sempre causam confusão: lembrar que a org é responsável por definir se pessoas que ministrarão conteúdos pagam ingressos e se organização paga ingresso, e que isso normalmente é definido com base nos orçamentos e arrecadação de patrocínios.

Reviewer: Boa, acho válido ingressos entrar no time de financeiro com tópico separado com essas considerações que vc fez.

Mto boa a ideia de doação de grupos subrepresentados, acho que vale um sub topico do topico ingressos. Pq aqui não precisamos discutir como fazer e sim levantar a possibilidade de ser feita.

Nooossaa verdade, esse último tópico sempre gera treta. Colocar um subtopico dos ingressos de "Definir se org paga ingressos" e outro "Definir se palestrantes e ministrantes de tutoriais pagam ingressos (e deixar EXPLICITO no formulario de submissao de palestras)"

> - Porta-voz de relacionamento com os keynotes

Reviewer 2: Como keynotes tem toda uma logística, não acho que deveria estar com o time de conteúdo.
É muito rolê agendar viagem, reservar hospedagem.
Seria melhor ter uma pessoa só por conta disso (no máximo duas, não precisa ser um time)

Reviewer: Justo, mas ai seria bom colocar essa tarefa em algum dos outros times, tem alguma sugestao de qual?

Reviewer 2: Eu acho que essa tarefa está mais próxima do time de orçamentos do que conteúdos.

Reviewer: Faz sentido, de todos acho que ta mais próximo de orçamentos msm.

  

## Gestão e organização

Uma vez feito a descrição dos times defina uma agenda mensal/quinzenal/semanal conforme o evento se aproxima. 
Irei sugerir aqui uma ordem de trabalho mas não existe regra essa ordem esta sempre em discução e deve ser sempre atualizada:

> Atividades:

Reviewer 2: Pra mim essas atividades deveriam ter uma ordem de prioridade e falta uma definição básica do evento nesse roadmap.
Acho que a primeira coisa a ser feita a nível de gestão é definir seu objetivo. Um evento pra quantas pessoas? Mais ou menos em qual época? Dá tempo de organizar até essa data?
A partir disso, fechar o local do evento e datas.
É necessário um orçamento mínimo para ter ideia dos custos do evento e quanto será necessário para viabilizá-lo. Isso vai ajudar a definir as cotas de patrocínio e também os preços de ingressos.
Minha sugestão de orçamentos básicos é local do evento + coffee break + audiovisual, porque eles variam muito de cidade pra cidade, os demais podem ser estimados.
Definir o plano de patrocínio é a próxima coisa que precisa ser feita (você precisa das informações básicas de onde e quando pra isso) e captar recursos é a prioridade máxima enquanto o evento não se tornar viável.
Paralelamente, levar as outras iniciativas: ingressos, orçamentos, conteúdos.
Ter um cronograma de ingressos e conteúdos é extremamente útil, e um deadline para orçamentos.

  - Coffe break
  - Audio visual:
    - Projetores
    - Microfones

Feito essa cotação inicial você deve ter material suficiente para fazer os planos de patrocinio, algo como 50%, 30%, 20%, 10% com base nos custos isso te dara o
diamante, ouro, prata, bronze, apartir disso você pode criar outras cotas lembrando que esse calculo é uma sugestão tudo pode depender da realidade do momento.

A proposta de roadmap abaixo sugere que seja feito a identidade visual antes do plano, mas as atividades podem ser paralelizadas. Outra sugestão é sempre focar primeiro no plano depois na data e local, mas essas atividades podem ser invertidas para o bem de um plano mais assertivo. 

Lembre-se o Plano de patrocinio não deve ser escrito em pedra, alterações podem e devem ser feitas rapidamente para se adaptar a situação socio-econômica da época. 

Empresas são pessoas só que mais burocráticas, pedindo com jeitinho e educação tudo se resolve até mudança de cotas, evite rebaixar empresas sempre suba as cotas ou crie algo maior, mas nunca rebaixe.

## Protótipo de roadmap

> Sugestão de trabalhos, as atividades e a ordem estão sempre em discussão e devem ser sempre atualizadas:

Reviewer 3: ```suggestion
> Sugestão de trabalhos, as atividades e a ordem estão sempre em discussão e devem ser sempre atualizadas:
```

Atividades:
 - Definir calendario de reuniões (mensal, quinzenal, semanal)

>  - Definir tamanho do evento:
>   - Numero de trilhas
>   - Numero de pessoas
>   - Numero de tutoriais
>   - Numero de locais (Muitas vezes não é possivel achar um espaço que contemple tutorias e palestras)

Reviewer 3: No site não foi suficiente essa quantidade de indentação.

![image](https://github.com/pythonbrasil/manual/assets/21251472/14abdc28-696a-4df4-980d-636b9986c546)

```suggestion
 - Definir tamanho do evento:
    - Numero de trilhas
    - Numero de pessoas
    - Numero de tutoriais
    - Numero de locais (Muitas vezes não é possivel achar um espaço que contemple tutorias e palestras)
```

- A grade do evento convida pessoas a vir, tente dar pelo menos 45 dias para as pessoas verem a grade e descidir se virão ou não, dependendo do local que o evento esta. Proximo aos extremos do pais, isso é muito importante.

- Chegando proximo ao evento concentre-se nos beneficios dos patrocinadores e em produzir todos os beneficios fisicos, exitem beneficios que precisam de entorno de 30 dias de antecipação como estandes, camisas, copos, alguns mais rapidos são as impressões como crachá, adesivos.

- O evento é muito importante para alunos de universidades, faculdades e escolas de ensino medio, busque encontrar acesso a essas pessoas e tente viabilizar a ida deles com apoio das escolas, caso o evento esteja financeiramente bom gaste tudo que puder para trazer novos integrantes para a comunidade sendo estudandes de escolas ou grupos subrepresentados, busque associaçoes regionais que possam lhe auxilar.

- A experiencia do evento deve correr sempre em paralelo mas no estagio final do evento isso sera muito necessario para finalizar compras (kit de banheiros, presente para palestrantes, keynotes, instrutores) e a organização de atividades (pybar, algohave, django-girls...).

> # Tretas comuns:

Reviewer 3: Para manter o padrão de cabeçalho 😄 

```suggestion
# Tretas comuns
```

Reviewer 3: Outro ponto, adiciona essa seção em um arquivo a parte. Acho que deixaria a usabilidade do site melhor, já que seria criado uma página nova com indices e tudo mais.

> #### Organização, Palestrantes, Ministrantes E os Ingressos
> ---
> Os organizadores, palestrantes, ministrantes de tutorial, pagam ingresso? Todo ano é discutido esse tema para cada um desses grupos
>  * Definam se organização paga ingresso e principalmente quem será considerado "organização". Uma opção é considerar que todas as pessoas que forem trabalhar no evento presencial não poderão aproveitá-lo, pois estarão ocupadas com as responsabilidades de org, então seria bom não pagarem.
>  * Definam isso para palestrantes e ministrantes de tutoriais ANTES de abrir o Call For Pappers para já deixar explicito na submissão.
>  * Dependendo da situação econômica da região, do pais e da verba do evento, essa questão pode ser reconsiderada. 
> 
> Deve-se deixar isso explicito a todos, pois o evento não tem orçamento proprio ele deve ser sempre autocontido mesmo as camisetas da organização saem do custo do evento.

Reviewer 3: ```suggestion
#### Os organizadores, palestrantes, ministrantes de tutorial, pagam ingresso?
---
Essa decisão é tomada pela organização. Dessa forma, todo ano é discutido esse tema para cada um desses grupos.

Levem em consideração os seguintes pontos:
 * Definam se organização paga ingresso e principalmente quem será considerado "organização". Uma opção é considerar que todas as pessoas que forem trabalhar no evento presencial não poderão aproveitá-lo, pois estarão ocupadas com as responsabilidades de org, então seria bom não pagarem.
 * Definam isso para palestrantes e ministrantes de tutoriais ANTES de abrir o Call For Papers para já deixar explicito na submissão.
 * Dependendo da situação econômica da região, do pais e da verba do evento, essa questão pode ser reconsiderada. 

> Deve-se deixar isso explicito a todos, pois o evento não tem orçamento proprio ele deve ser sempre autocontido mesmo as camisetas da organização saem do custo do evento.
```

> ####  Codigo de conduta (CDC)
> ---
> O comite de codigo de conduta é organizado e montado com ajuda da organização, mas é totalmente independente tendo poder até em cima da organização durante a realização do evento. 
> 
> O CDC só cobre as atividades oficiais da python brasil (tutoriais, sprints, palestras, django-girls), todos os demais eventos extraofficiais (algorave, pybar, pos-evento) pode causar situações para os proximos dias mas o CDC não atua fora do evento (possiveis situações devem ser tratadas no evento). 
> 
> A comunidade se posiciona sempre e ajuda em outras situações, mas a organização do evento não pode se comprometer com isso.

Reviewer 3: ```suggestion
####  Código de conduta (CDC)
---
O Comitê de Código de Conduta é organizado e montado com ajuda da organização, mas é totalmente independente tendo poder até em cima da organização durante a realização do evento. 

O CDC só cobre as atividades oficiais da Python Brasil (tutoriais, sprints, palestras, Django Girls), todos os demais eventos extraoficiais (Algorave, PyBar, Pós-Evento) pode causar situações para os próximos dias, mas o CDC não atua fora do evento (possíveis situações devem ser tratadas no evento). 

A comunidade se posiciona sempre e ajuda em outras situações, mas a organização do evento não pode se comprometer com isso.

> Importante destacar que é imprescindível, que toda a organização e comitê estejam a par do conteúdo do [Código de Conduta](https://python.org.br/cdc/).
```

> #### Apoio a grupos subrepresentados
> ---
> 
> Programas de apoio de custo e ingressos cortezia para grupos subrepresentados: esse tipo de programa é sempre bem vindo, mas a organização deve sempre olhar para o financeiro e a viabilização do evento. 
> 
> O evento estando alinhado pode-se ir atrás desses grupos e viabilizar a vinda deles atravez de iniciativas dos próprios grupos com o apoio da organização (apoiar na divulgação, alcance, e suporte no sentido de centralizar com o grupo "X").
> 
> A organização não deve se comprometer com a criação desses programas, pois é uma atividade da comunidade. A organização sempre é e provavelmente sempre vai ser sobrecarregada com outras atividades.
> 
> 
> **OBS**: Cuidado para não abrir precedentes que possam exigir que toda a organização faça pois a comunidade gostou.

Reviewer 3: ```suggestion
#### Programas de apoio de custo e ingressos cortesia para grupos sub-representados
---

Esse tipo de programa é sempre bem vindo, mas a organização deve sempre olhar para o financeiro e a viabilização do evento. 

A organização estando definida e organizada, pode-se ir atrás desses grupos e viabilizar a vinda deles através de iniciativas dos próprios grupos com o apoio da organização (apoiar na divulgação, alcance e suporte para viabilizar a parceria).

> A organização não deve se comprometer com a criação desses programas, pois é uma atividade da comunidade. A organização sempre é, e provavelmente sempre vai ser, sobrecarregada com outras atividades.

**OBS**: Cuidado para não abrir precedentes que possam exigir que toda a organização faça pois a comunidade gostou.
```
